packages/brand-components/src/Avatar/index.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import s from './styles.module.scss';
+import cx from 'classnames';
+import { VisualTheme } from '../../../gamut/src/theming/VisualTheme';
+
+type imageProps =
```

Comment:
These are generally done in PascalCase (and with a bit more information IMO):

```suggestion
type AvatarImageProps =
```

Reply:
_(same with, say, `AvatarBaseProps`?)_